docs/docs/index.md

```diff
@@ -42,7 +42,7 @@ You should now be seeing something like this:
 The upper part of the window is "divided" into two sections:
 
 * The search form: to make searches using tags. To insert a date, or access advanced features, you can click the "Plus" button. You can also browse other pages by changing the page value on the right.
-* Hidden form: you can show it by clicking the "+" button. Here, you have some fields to quickly change the settings (images per page and columns), the sources and "Post-filtering". Post-filtering allows you, for example, to bypass the two tags limitation of Danbooru by adding additional tags or metas (source, id, height, etc.) in this field. Note that none of these fields will be saved at exit, so if you want to definitely change the number of images per page or columns for example, you have should change them directly in the options.
+* Hidden form: you can show it by clicking the "+" button. Here, you have some fields to quickly change the settings (images per page and columns), the sources, and "Post-filtering". Post-filtering allows you, for example, to bypass the two-tags limitation of Danbooru by adding additional tags or meta (source, id, height, etc.) in this field. Note that none of these fields will be saved at exit, so if you want to change the number of images per page or columns for example, you should change them directly in the options.
```

Comment:
I agree "definitely" sounds weird here, but it was pointing out that while changing the setting in that area is temporary, if the user wants the change to remain for future searches / other tabs / after a restart, it should be changed in the settings instead. Maybe something like "so if you want Grabber to remember your changes to the number of images per page or columns for example, [...]" would work?

Reply:
Your suggestions sound better than the current documentation, I'll implement the changes and see how it'll go.